textattack/shared/utils.py

```diff
@@ -109,6 +109,13 @@ def default_class_repr(self):
         extra_str = ''
     extra_str = extra_str.format(**self.__dict__)
     return f'{self.__class__.__name__}{extra_str}'
+
+def color_text(label, color=None, method=None):
```

Comment:
Why did you rename this to color_text? Maybe color_label isn't the best name if we're going task-agnostic but I feel like having a color_text and a color_text_by_method for which the color_text changes some stuff then calls color_text_by_method is confusing.

Reply:
thanks! I consolidated these into a single method, `color_text`